grafana/overlays/nerc-ocp-obs/grafanas/grafana.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: integreatly.org/v1alpha1
+kind: Grafana
+metadata:
+  name: grafana
+  namespace: grafana
+spec:
+  deployment:
+    envFrom:
+      - secretRef:
+          name: oauth-client-secret
+      - secretRef:
+          name: grafana-serviceaccount-token
+      - configMapRef:
+          name: openshift-service-ca.crt
+  config:
+    server:
+      root_url: https://grafana.apps.nerc-ocp-obs.rc.fas.harvard.edu
+    auth.generic_oauth:
```

Comment:
The url is wrong; the canonical url for the observability cluster is `obs.nerc.mghpcc.org`. So this should be `https://grafana.apps.obs.nerc.mghpcc.org`.